### PR TITLE
Sort all the things alphabetically

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Comment by callunity** [comment]
Please also change the sampleClass sort in the EditSampleController (~L418)

**Comment by apmasell** [comment]
Fixed

File: miso-web/src/main/webapp/scripts/sample_hot.js [hunk 1]
```diff
@@ -220,10 +225,17 @@ Sample.hot = {
   },
 
   /**
+   * Returns true if a new sample of the provided SampleClass can be created without an existing parent
+   */
+  canCreateNew: function (sampleClass) {
```
Comment (djcooke) [inline miso-web/src/main/webapp/scripts/sample_hot.js]: Shouldn't this now include some (not all) aliquots? sampleClass.canCreateNew probably replaces the need for this function though

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/sample_hot.js]: Oh, this was a bad rebase. I'll remove it.

**Pull request merged**